external/external.sh

```diff
@@ -285,14 +285,15 @@ parseCommandLineArgs "$@"
 if [ $command_type == "prepare" ]; then
 	# Specify docker.io or internal registry to prepare base image with login to increase pull limit or authenticate; Redhat Registry no login.
 	if [[ $base_docker_registry_url != "default" ]]; then
-		# Container credential check. Temporarily criu-ubi image with criu binary is only available internally
-		if [[ ! -z $BASE_DOCKER_REGISTRY_CREDENTIAL_USR || ! -z $DOCKER_REGISTRY_CREDENTIALS_USR ]]; then
+		# Container credential check. 
+		# Temporarily criu-ubi image with criu binary is only available internally
```

Comment:
This comment is a bit hard to read :-).  Not quite sure what logic it is trying to convey.

Reply:
In this step, external tests get the initial base container image. Most tests uses `BASE_DOCKER_REGISTRY_CREDENTIAL` to get this base image from Dockerhub (this login will increase pull limit), but `criu-ubi` base image temporarily get its base image from hyc-dockerhub instead since we can only set up its dependent package internally with ubi subscription. Once we are able to publish this image on Dockerhub later, we will remove this logic then.